docs/pages/enroll-resources/database-access/enroll-azure-databases/azure-sql-server-ad.mdx

```diff
@@ -66,6 +66,9 @@ Teleport requires the following permissions:
 - `<resource-type>/read` permissions for discovery, but only for the resource
   types you have. For example, `Microsoft.Sql/managedInstances/read`.
 - `Microsoft.Compute/virtualMachines/read`.
+- If your Database Service runs on a Virtual Machine Scale Set (VMSS) in uniform
+  mode, such as on an AKS cluster. In that case, `Microsoft.Compute/virtualMachineScaleSets/virtualMachines/read`
+  permission will also be required.
```

Comment:
could you also add some context to be line above `Microsoft.Compute/virtualMachines/read`

```suggestion
- `Microsoft.Compute/virtualMachines/read`. <-- add some context 
- `Microsoft.Compute/virtualMachineScaleSets/virtualMachines/read` permission if your Database Service runs on a Virtual Machine Scale Set (VMSS) in uniform mode, such as on an AKS cluster.
```

Also do we need to update the description next line where it says `read **Azure SQL Servers**
and **Azure SQL Managed Instances**`?

Reply:
I've done a minor rework on this section. Let me know what you think.